src/python/pants/backend/python/dependency_inference/module_mapper_test.py

```diff
@@ -779,3 +901,40 @@ def test_issue_15111(rule_runner: RuleRunner) -> None:
             }
         )
     )
+
+
+def test_generate_mappings_from_pattern_matches(rule_runner: RuleRunner) -> None:
+    assert generate_mappings_from_pattern("google-cloud-hardyhar", "bar") == [
```

Comment:
**comment:** "bar"` is the fallback and will always be output, at a minimum. see a previous [comment ](https://github.com/pantsbuild/pants/pull/19684/files#r1309078101) on thoughts about including the fallback.